docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -496,30 +496,110 @@ A cicd_task is a single job of ci/cd.
 | `finished_date` | datetime        | 3          | when did this task finish                                       |         |
 | `environment`   | varchar         | 255        | To indicate the environment in which the task is running        |         |
 
+### Domain 5 - Code Quality

```

Comment:
We can add a line here: The names of tables in the 'Code Quality' domain will start with a prefix `cq_`